src/cfnlint/data/ExtendedSpecs/all/03_value_types/aws_appflow.json

```diff
@@ -0,0 +1,82 @@
+[
+ {
+  "op": "add",
+  "path": "/ValueTypes/AWS::AppFlow::Connector.Description",
+  "value": {
+   "StringMax": 1024,
```

Comment:
Need to check this one.  The schema defines 2048 and the docs have 1024

"Description": {
   "description": "A description about the connector that's being registered.",
   "maxLength": 2048,
   "pattern": "[\\s\\w/!@#+=.-]*",
   "type": "string"
  }